NwPluginAPI/Enums/ServerEventType.cs

```diff
@@ -924,5 +926,25 @@ public enum ServerEventType : int
 		/// Cancellable with <see cref="RoundEndConditionsCheckCancellationData"/>.
 		/// </remarks>
 		RoundEndConditionsCheck = 111,
+
+		/// <summary>
+		/// Executed after a pickup is upgraded by the SCP-914.
+		/// </summary>
+		/// <remarks>
+		/// Parameters: <see cref="ItemPickupBase"/> item, <see cref="Vector3"/> newPosition
+		/// </remarks>
+		/// <param name="item">The upgraded item</param>
+		/// <param name="newPosition">The position where the item will be moved to</param>
```

Comment:
I don't know if it bothers you but I wanted to try adding this data in the summary since it might be useful, besides when someone sees the ServerEventType.ThisEvent it looks like this 
![image](https://user-images.githubusercontent.com/36207738/210006556-38f9424b-64af-41b1-a0f7-731885aaf442.png)
and I think it looks good

Reply:
![2022-12-29_21-47-03](https://user-images.githubusercontent.com/7418075/210009642-d029b099-73fc-478f-a724-816092158570.png)

That's invalid - you can't reference a param that doesn't exist. Of course some IDEs will show it that way, but it's invalid and some IDEs might not display it.

Reply:
Fixed with commit [99e28e4](https://github.com/northwood-studios/NwPluginAPI/pull/123/commits/99e28e4ae6e7afaf6d8e70d5b602be91ab9c93cd)